jboss/container/jolokia/bash/module.yaml

```diff
@@ -31,9 +31,9 @@ execute:
 - script: backward_compatibility.sh
 
 artifacts:
-#- name: jolokia-jvm-agent.jar
-- url: https://maven.repository.redhat.com/ga/org/jolokia/jolokia-jvm/1.5.0.redhat-1/jolokia-jvm-1.5.0.redhat-1-agent.jar
-  md5: d31c6b1525e6d2d24062ef26a9f639a8
+- target: jolokia-jvm-1.6.2.redhat-00002-agent.jar
+  name: jolokia-jvm-1.6.2.redhat-00002-agent.jar
```

Comment:
Pro-tip: specify `name` independent of the `target`: for example: `jolokia_jar`, so that it will be easy to override it.

Reply:
Related: https://github.com/cekit/cekit/issues/573